pyspecProcScripts/correlation_alignment.py

```diff
@@ -199,16 +222,17 @@ def correl_align(
             )
         else:
             f_shift = correl.run(np.real).argmax(direct)
-        s_copy = s_orig.C
-        s_copy *= np.exp(-1j * 2 * np.pi * f_shift * s_copy.fromaxis(direct))
-        s_orig.ft(direct)
-        s_copy.ft(direct)
+        # Calculate energy with shift applied
+        s_aligned = s_jk * np.exp(
+            -1j * 2 * np.pi * f_shift * s_jk.fromaxis(direct)
+        )
+        s_aligned.ft(direct)
         if my_iter == 0:
             logging.debug(psd.strm("holder"))
             if fl:
-                s_copy.reorder([direct], first=False)
+                s_orig.reorder([direct], first=False)
```

Comment:
here, it does make sense to not reorder the original -- so I don't understand this.

Reply:
nixed this